go.mod

```diff
@@ -21,5 +21,6 @@ require (
 	github.com/rs/zerolog v1.19.0
 	github.com/stretchr/testify v1.6.1
 	google.golang.org/genproto v0.0.0-20200806141610-86f49bd18e98
+	gopkg.in/yaml.v2 v2.2.7
```

Comment:
Do you know why we depend on it now? I don't see anything YAML-related.

Reply:
https://github.com/athenianco/cloud-common/pull/7#discussion_r648340445

Reply:
Still, do we use YAML there?

Reply:
found the root cause, fixed